paymentsheet/src/main/java/com/stripe/android/paymentsheet/navigation/PaymentSheetScreen.kt

```diff
@@ -417,4 +426,50 @@ internal sealed interface PaymentSheetScreen {
             ManageOneSavedPaymentMethodUI(interactor = interactor)
         }
     }
+
+    class CvcRecollection(
+        private val args: CvcRecollectionContract.Args,
+        private val interactor: CvcRecollectionInteractor
+    ) : PaymentSheetScreen, Closeable {
+        override val showsBuyButton = false
```

Comment:
I think based on the video in [this slack](https://stripe.slack.com/archives/C06JTLU0K8E/p1721157118982699?thread_ts=1719511478.480099&cid=C06JTLU0K8E) this screen _should_ show the buy button, and then the user can just pay from here